docs/UserGuide.md

```diff
@@ -273,21 +351,45 @@ Examples:
 * `list` followed by `clear` clears all cases from DengueHotspotTracker.
 * `find n/Betsy` followed by `clear` deletes all cases with names that include "Betsy".
 
+<div markdown="block" class="alert alert-primary">
+
+**:exclamation: Warning:**<br>
+
+As `clear` resets all the data in `DengueHotspotTracker` you should use [`delete`](#deleting-cases-delete) for use cases of deleting specific or groups of cases.
+
+</div>
+
 <br>
 
 ### Listing all cases: `list`
 
-Displays all cases in DengueHotspotTracker.
+Use this command to reset the find filters and display all cases in DengueHotspotTracker.
 
 Format: `list`
 
+<div markdown="block" class="alert alert-primary">
+
+**:bulb: Tip:**<br>
+
+As `list` resets all active filters and sort orderings, one of the most common uses of the `list` command to reset the display lists to display all the properties and buyers in `DengueHotspotTracker`.
+Some places where you might want to make use of the `list` are when:
+* [`delete`](#deleting-cases-delete) to show all the cases as delete follows the index of the current list
```

Comment:
Might be better to discuss `clear` here instead? Think the use case for listing before deleting a particular index (since delete follows the index of the current list, one can delete a case whenever they wish as long as it's in the current shown list) is not as obvious as the use case for listing before clearing (want to clear **all** cases).

Reply:
If not: it would be great to update the phrasing here to match the phrasing of overview and clear! ("which ____").

Reply:
can be done in future iteration